pkg/proxy/ipvs/proxier.go

```diff
@@ -431,22 +432,47 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	return mods, nil
 }
 
+// CheckBuiltinModules checks whether pasticular modules are built-in or not.
+func (handle *LinuxKernelHandler) CheckBuiltinModules(mods []string) (bool, error) {
+	knernelVersion, err := handle.executor.Command("uname", "-r").CombinedOutput()
```

Comment:
s/knernelVersion/kernelVersion/

Reply:
Would it be better to use `syscall.Uname` or pull this from `/proc/sys/kernel/osrelease` rather than shelling out to `uname`?